framework/archiver/listeners.py

```diff
@@ -0,0 +1,41 @@
+from framework.tasks.handlers import enqueue_task
+from framework.archiver.tasks import archive, send_success_message
+from framework.archiver.utils import (
+    link_archive_provider,
+)
+from framework.archiver import (
+    ARCHIVER_SUCCESS,
+    ARCHIVER_FAILURE,
+)
+from framework.archiver.exceptions import ArchiverCopyError
+
+from website.project import signals as project_signals
```

Comment:
We should talk in person about the level of coupling/architecture concerns expressed in some of these comments. 

My intention is to keep archiver as decoupled from website code as possible. Perhaps a better pattern is import archiver functionality as needed into website code, and to ```@connect``` to this signal therein.